<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runtime_control</class>
 <widget class="QWidget" name="runtime_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>186</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>基础设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/ico/ico_file/yunxingshi.png</normaloff>:/ico/ico_file/yunxingshi.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_output_setup">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>5000</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>门控</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>延时</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboBox_delay_d">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ns</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>s</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_delay_d">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>脉宽</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_width_d">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="comboBox_width_d">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ns</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>s</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>输出A</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_channel_enable_a">
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>延时</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_delay_a">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>脉宽</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_width_a">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboBox_delay_a">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ns</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>s</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="comboBox_width_a">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ns</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>s</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>输出B</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox_channel_enable_b">
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_delay_b">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>脉宽</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_width_b">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>延时</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="comboBox_delay_b">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ns</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>s</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="comboBox_width_b">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ns</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>s</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>输出C</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_channel_enable_c">
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>延时</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_delay_c">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>脉宽</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_width_c">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboBox_delay_c">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ns</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>s</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="comboBox_width_c">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ns</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>s</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_exposure">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>1000</height>
        </size>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>0.000250000000000</double>
       </property>
       <property name="maximum">
        <double>30.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>曝光时间(S)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>MCP增益(0-4095)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_mcp_gain">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>1000</height>
        </size>
       </property>
       <property name="maximum">
        <number>4095</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget_output_setup</tabstop>
  <tabstop>doubleSpinBox_delay_d</tabstop>
  <tabstop>comboBox_delay_d</tabstop>
  <tabstop>doubleSpinBox_width_d</tabstop>
  <tabstop>comboBox_width_d</tabstop>
  <tabstop>doubleSpinBox_exposure</tabstop>
  <tabstop>spinBox_mcp_gain</tabstop>
  <tabstop>checkBox_channel_enable_a</tabstop>
  <tabstop>doubleSpinBox_delay_a</tabstop>
  <tabstop>comboBox_delay_a</tabstop>
  <tabstop>doubleSpinBox_width_a</tabstop>
  <tabstop>comboBox_width_a</tabstop>
  <tabstop>checkBox_channel_enable_b</tabstop>
  <tabstop>doubleSpinBox_delay_b</tabstop>
  <tabstop>comboBox_delay_b</tabstop>
  <tabstop>doubleSpinBox_width_b</tabstop>
  <tabstop>comboBox_width_b</tabstop>
  <tabstop>checkBox_channel_enable_c</tabstop>
  <tabstop>doubleSpinBox_delay_c</tabstop>
  <tabstop>comboBox_delay_c</tabstop>
  <tabstop>doubleSpinBox_width_c</tabstop>
  <tabstop>comboBox_width_c</tabstop>
 </tabstops>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_channel_enable_a</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_delay_a</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_a</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_width_a</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_a</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_delay_a</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_a</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_width_a</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_b</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_delay_b</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_b</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_width_b</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_b</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_delay_b</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_b</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_width_b</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_c</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_delay_c</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_c</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_width_c</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_c</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_delay_c</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_channel_enable_c</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_width_c</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
